tox.ini

```diff
@@ -26,7 +26,7 @@ commands =
 deps =
     doc8
     pyenchant
-    readme>=0.5.1
+    readme_renderer>=0.5.1
```

Comment:
There's only an 0.7 release of this, so the condition doesn't make sense and can be dropped.